examples/example.go

```diff
@@ -0,0 +1,179 @@
+package examples
+
+import (
+	"crypto/tls"
+	"fmt"
+	"math/rand"
+	"net/http"
+	"time"
+
+	"github.com/comvex-jp/uipath-go"
+	"github.com/comvex-jp/uipath-go/configs"
+	"github.com/patrickmn/go-cache"
+)
+
+type Examples struct {
+	Client *uipath.Client
+}
+
+func Run() {
+	tr := &http.Transport{
+		TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
+	}
+
+	e := &Examples{
+		Client: &uipath.Client{
+			HttpClient: &http.Client{Transport: tr},
+			Credentials: uipath.Credentials{
+				ClientID:   "{{test_client_Id}}",
+				UserKey:    "{{test_user_key}}",
+				TenantName: "{{test_tenant_name}}",
+			},
+			BaseURL: "{{test_base_url}}",
+			Cache:   cache.New(5*time.Minute, 10*time.Minute),
+		},
+	}
+
+	res, err := uipath.GetOAuthToken(e.Client)
+	if err != nil {
+		fmt.Println(res, err)
+	}
```

Comment:
This is just an example but we don't need this anymore every time we want to execute a handler right? Maybe we can remove this here then?